docs/development/testing.md

```diff
@@ -135,18 +139,18 @@ Mocha does not automatically compute code coverage. If you need it, use:
 - [nyc][]: the most popular code-coverage tool; the successor CLI for Istanbul
 
 [`assert` module]: https://nodejs.org/api/assert.html#assert_assert
-[Babel]: https://babeljs.io
-[Chai]: https://www.npmjs.com/package/chai
+[babel]: https://babeljs.io
```

Comment:
Ok.. So I have run lint but apparently, that doesn't seem to work well with this style of links
Fixing it manually - there were multiple lint issues checked on CI/CD